### DISCUSSION TITLE
JK-BMS Bluetooth

### DISCUSSION THREAD

**Discussion by WagnerSystems** [shining-man/bsc_fw#40] [General]
Wird es eine Unterstützung des JK-BMS über Bluetooth geben?

**Comment by shining-man** [comment]
Ich habe das JK-BMS (Bluetooth) bereits zu Testen implementiert.
Sollte im Branch New-features enthalten sein. 
Aktuelle Testversionen findest du auch auf dem Discord-Server zum BSC.
Einladung: https://discord.gg/sEvwS9zU
